### PR TITLE
Support multiple origin names

### LINKED ISSUE #108: Consider including multiple origin names in TokenChallenge
From Ted Hardie in Vienna.

### PR DESCRIPTION
**Pull request by tfpauly**
Closes #108

### CODE REVIEW

File: draft-ietf-privacypass-auth-scheme.md [hunk 1]
```diff
@@ -186,10 +187,14 @@ WWW-Authenticate: PrivateToken challenge=abc..., token-key=123...
 Upon receipt of this challenge, a client uses the message and keys in the
 issuance protocol indicated by the token_type. If the TokenChallenge has a
 token_type the client does not recognize or support, it MUST NOT parse or
-respond to the challenge.
+respond to the challenge. If the TokenChallenge contains a non-empty
+origin_info field, the client SHOULD validate that the name of the origin
+that issued the authentication challenge is included in the list of origin
+names. Clients MAY have further restrictions and requirements around
+validating when a challenge is considered acceptable or valid.
```
[the remark this answers is not in the document]
Comment (chris-wood) [reply draft-ietf-privacypass-auth-scheme.md:194]: Also, in moving to a MUST, we should add text to the security considerations which explain why this effectively means origins must "cross sign" each other (by using the same origin_info) in order for tokens to be spent across origins.

**Review by chris-wood — CHANGES_REQUESTED** [review]

File: draft-ietf-privacypass-auth-scheme.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (tfpauly) [reply draft-ietf-privacypass-auth-scheme.md]: @chris-wood added text to security considerations, please check

**Review by chris-wood — APPROVED** [review]
🎉 perfect!

**Comment by chris-wood** [comment]
@dvorak42 please have a look!

**Review by dvorak42 — APPROVED** [review]

**Pull request merged**